sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -331,9 +331,9 @@ export class EventHubConsumerClient {
   /**
    * Subscribe to all messages from a subset of partitions.
    *
-   * Use this overload if you want to read from a specific set of partitions and not coordinate with
+   * Use this overload if you want to read from a specific partition and not coordinate with
```

Comment:
What does `not coordinate with other subscribers` mean?

Reply:
Removing. The doc comments drifted and became inaccurate :)